Modules/Filtering/Path/include/itkPath.h

```diff
@@ -126,9 +126,9 @@ class ITK_TEMPLATE_EXPORT Path : public DataObject
   itkGetConstMacro(ZeroIndex, IndexType);
 
 private:
-  // These "constants" are initialized in the constructor
-  OffsetType m_ZeroOffset{}; // = 0 for all dimensions
-  IndexType  m_ZeroIndex{};  // = 0 for all dimensions
+  // These constants are initialized at compile time
+  const OffsetType m_ZeroOffset{}; // = 0 for all dimensions
+  const IndexType  m_ZeroIndex{};  // = 0 for all dimensions
```

Comment:
Even though those data members are `const`, they are still initialized in the (defaulted) constructor. If you want them to be initialized at compile time, declare them `static constexpr`.

Reply:
Eventually I hope we can just remove those data members, as well as those member functions, Path::GetZeroOffset() and Path::GetZeroIndex(). I think a member function that always just returns a zero-initialized Offset, or a zero-initialized Index isn't really necessary, right?  🤷 Anyway, that may be another pull request  😺

Reply:
Let me see if I have this correct: The Index<> and Offset<> types are both POD.  So POD member variable with explicit default initialization, will not get initialized to zeros. But a static constexpr POD member variable will get initialized to zero with a compile file value?

Or are you saying that with this proposed close those variable are not initialize. Please be specific with your recommended changes.